### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [eleferrets/student_store#1]
Thank you for submitting the **Student Store** assignment! We have graded your work, and it looks like the following features are not reflected on your GIF walkthrough:

- The API should contain an endpoint that serves an array of all products in the store
- The website is deployed using Heroku and Surge.
  - The deployed website does not show the products. 

You may refer to this [sample video](https://youtu.be/lpOr3Iligbs) on how to submit your assignment.

In order for us to count these towards your submission, please record another GIF that captures these features. Once you do, please push your updates and submit your assignment again through the course portal within 48 hours from the posted deadline so that we can regrade it. Please note that this will count toward one of your 3 allowable 48 hour extensions to submit an assignment. Reach out with any questions or concerns!

**Comment by eleferrets** [comment]
Good evening,
Thank you for your response, but I wanted to clarify some things for the
assignment. For the store model, I did include a store model on the
backend, but it did not say anything about having a cart or adding items to
the cart in the required section. I did have a store model that did make
read and writes to the JSON file in order for the products to display, but
I believe having a cart was stretch feature.

For the heroku deployment, I put in a note that the react frontend works,
but the products do not display. I followed the instructions for the
deployment instructions and the application makes a request to the database
without errors, but things do not display. If things do not work out, I am
okay with leaving out the deployment as a completed stretch feature.

For the video walkthrough, I believe I put in all of the required features
with having a store model, displaying all the products on the non-deployed
app, correct routing, and having an individual product page.

Since I had all the required features done, may I ask why one of the
extensions are being used? Thank you for your time and have a nice night.
